airflow/models/skipmixin.py

```diff
@@ -98,3 +98,17 @@ def skip_all_except(self, ti: TaskInstance, branch_task_ids: Union[str, Iterable
 
             self.log.info("Skipping tasks %s", [t.task_id for t in skip_tasks])
             self.skip(dag_run, ti.execution_date, skip_tasks)
+
+    def evaluate_skip_condition(self, xcom_value: Any, downstream_ti: TaskInstance):
+        """
+        Evalutes the xcom_value of the SkipMixin itself and decides if a downstream task
+        should be skipped.
```

Comment:
This doc needs expanding to say when this is called (as it's not clear without reading the rest of this code) so people with custom operators know what to do.